zyte_spider_templates/heuristics.py

```diff
@@ -56,3 +63,43 @@ def might_be_category(url: str) -> bool:
                 return False
 
     return True
+
+
+INDEX_URL_PATHS = {
```

Comment:
Do we plan to improve this list later?

Reply:
Yeah, we can update it as we encounter new ones.